lib/HLSL/DxilContainerReflection.cpp

```diff
@@ -2837,59 +2847,66 @@ HRESULT CFunctionReflection::GetDesc(D3D12_FUNCTION_DESC *pDesc) {
-  // used Unset:  UINT                    DynamicFlowControlCount;     // Number
-  // of dynamic flow control instructions used Unset:  UINT
-  // MacroInstructionCount;       // Number of macro instructions used Unset:
-  // UINT                    ArrayInstructionCount;       // Number of array
-  // instructions used Unset:  UINT                    MovInstructionCount; //
-  // Number of mov instructions used Unset:  UINT MovcInstructionCount; //
-  // Number of movc instructions used Unset:  UINT ConversionInstructionCount;
-  // // Number of type conversion instructions used Unset:  UINT
-  // BitwiseInstructionCount;     // Number of bitwise arithmetic instructions
-  // used Unset:  D3D_FEATURE_LEVEL       MinFeatureLevel;             // Min
-  // target of the function byte code
+  //                                 instructions
+  // Unset: UINT TextureLoadInstructions; // Number of texture load
+  //                                         instructions
+  // Unset: UINT TextureCompInstructions; // Number of texture comparison
+  //                                         instructions
+
+  // Unset: UINT TextureBiasInstructions;// Number of texture bias instructions
+  // Unset: UINT TextureGradientInstructions; // Number of texture gradient
+  //                                             instructions
+  // Unset: UINT FloatInstructionCount; // Number of floating point arithmetic
+  //                                       instructions used
+  // Unset: UINT IntInstructionCount;   // Number of signed integer arithmetic
+  //                                       instructions used
+  // Unset: UINT UintInstructionCount; // Number of unsigned integer
+  //                                      arithmetic instructions used
+  // Unset: UINT StaticFlowControlCount;  // Number of static flow control
+  //                                         instructions used
+  // Unset: UINT DynamicFlowControlCount; // Number of dynamic flow control
+  //                                         instructions used
+  // Unset: UINT MacroInstructionCount; // Number of macro instructions used
+  // Unset: UINT ArrayInstructionCount; // Number of array instructions used
+  // Unset: UINT MovInstructionCount; // Number of mov instructions used
+  // Unset: UINT MovcInstructionCount; // Number of movc instructions used
+  // Unset: UINT ConversionInstructionCount; // Number of type conversion
+  //                                            instructions used
+  // Unset: UINT BitwiseInstructionCount; // Number of bitwise arithmetic
+  //                                         instructions used
+  // Unset: D3D_FEATURE_LEVEL MinFeatureLevel; // Min target of the function
+  //                                              byte code
 
   pDesc->RequiredFeatureFlags =
       m_FeatureFlags & ~(UINT64)D3D_SHADER_REQUIRES_EARLY_DEPTH_STENCIL;
+  // Also Mask off function-level derivatives flag.
```

Comment:
Why mask off function-level flag?

Reply:
It's a temporary flag that isn't meant to make it to the final feature flags as runtime has no need of it. It's used to determine derivative usage for the sake of cs/as/ms shaders. Once it has been used to determine what the entry points use, it can and should be discarded.